tests/arch/arm/arm_no_multithreading/testcase.yaml

```diff
@@ -6,3 +6,4 @@ tests:
   arch.arm.no_multithreading:
     filter: not CONFIG_TRUSTED_EXECUTION_NONSECURE
     platform_allow: qemu_cortex_m0 qemu_cortex_m3 mps2_an385 mps2_an521
+      nrf52840dk_nrf52840 nrf9160dk_nrf9160
```

Comment:
Can you please add `nrf51dk_nrf51422` ?

Reply:
@ioannisg can you test on the 51DK please?

Reply:
tested - works fine